### ISSUE TITLE
Play from url no longer works

### ISSUE THREAD

**Issue by villermen** [team-thyme/soundboard#98]
a.size is not a function()

```
}, {
                key: "playRandomWithId",
                value: function(e) {
                    var t = e.id
                      , n = e.scroll
                      , r = void 0 !== n && n
                      , o = e.addToHistory
                      , i = void 0 !== o && o
                      , a = (0,
                    s.default)(".sample").filter(function() {
                        return (0,
                        s.default)(this).data("sample").id === t
                    });
                    if (0 === a.size())
                        return null ;
                    var u = Math.floor(Math.random() * a.length)
                      , l = a.eq(u);
                    return l.trigger("click", {
                        addToHistory: i
                    }),
                    r && this.scrollToSample(l),
                    l
                }
            }, {
```

**Comment by villermen** [comment]
.size() is deprecated in the updated jQuery, so use .length.

**Comment by villermen** [comment]
Fixed in 6d6c7a711bf3927047c9e5b0721b47a314eb266a.
